projectDocs/dev/createDevEnvironment.md

```diff
@@ -62,20 +62,14 @@ To replicate the production build environment, use the 3.11.x minor version of P
 		* MSVC v143 - VS 2022 C++ x64/x86 build tools
 		* C++ ATL for v143 build tools (x86 & x64)
 		* C++ ATL for v143 build tools (ARM64/ARM64EC)
-* When you are using Visual Studio Code as your integrated development environment of preference, you can make use of our [prepopulated workspace configuration](https://github.com/nvaccess/vscode-nvda/) for [Visual Studio Code](https://code.visualstudio.com/).
-	While this VSCode project is not included as a submodule in the NVDA repository, you can easily check out the workspace configuration in your repository by executing the following from the root of the repository.
-
-	```sh
-	git clone https://github.com/nvaccess/vscode-nvda.git .vscode
-	```
 
 ### Git Submodules
 Some of the dependencies are contained in Git submodules.
 If you didn't pass the `--recursive` option to git clone, you will need to run `git submodule update --init`.
 Whenever a required submodule commit changes (e.g. after git pull), you will need to run `git submodule update`.
 If you aren't sure, run `git submodule update` after every git pull, merge or checkout.
 
-For reference, the following run time dependencies are included in Git submodules:
+#### Run time dependencies
```

Comment:
line 90 should probably be changed to `build time dependencies`

Reply:
I'm going to open another PR to fix this and tidy up the dev environment docs